app/controllers/terms_controller.rb

```diff
@@ -22,8 +22,8 @@ def create
     @vocabulary = find_vocabulary
     combined_id = CombinedId.new(params[:vocabulary_id], term_params[:id])
     term_form = term_form_repository.new(combined_id)
-    term_form.attributes = term_params.except(:id)
-    term_form.set_languages(params)
+    term_form.attributes = vocab_params.except(:id)
```

Comment:
So the way the data is being wrapped by simple form, 
```
{:term => { :term_id => :blah},
 :vocabulary => {
      :vocab_info => "blah",
       :language => {:vocab_info => "banana"}
}
```
Meant that we needed a second method to handle cleaning both term and vocabulary. Term is only meant to contain the term id and the rest is stuffed in vocabulary. So yes they are term params, but in all reality, they are actually still "vocab" params. Since thats where the data is stored.